tools/tokenserver/purge_old_records.py

```diff
@@ -336,17 +341,21 @@ def main(args=None):
     )
 
     opts, args = parser.parse_args(args)
-    if len(args) != 2:
+
+    if len(args) == 0:
         parser.print_usage()
         return 1
 
-    secret = args[1]
+    # Secret is the last arg?
+    secret = args[-1]
```

Comment:
I don't understand this change, was it necessary (as this only takes one positional `secret` arg)?

Reply:
My reasoning is:

We were pulling `arg[1]` and checking that we had two args. This meant that the `usage` [message](https://github.com/mozilla-services/syncstorage-rs/blob/master/tools/tokenserver/purge_old_records.py#L253) was wrong, because `purge_old_records.py  secret` would not work. 

`optparse.OptionsParser()` returns the `args` as a list (e.g. `args = ["secret"]`) meaning that for the script to work correctly, we would have to introduce another argument that we are ignoring. Since `args[1]` equates to the last of the two arguments being passed, it's also equivalent to being the last argument being passed, since previously, we were enforcing that [there be two arguments](https://github.com/mozilla-services/syncstorage-rs/blob/master/tools/tokenserver/purge_old_records.py#L339) passed. 

This fix would allow us to either specify the existing two arguments, or drop the first, ignored argument and still function. 

Is that mistaken?

Reply:
Got it, sorry I was misreading it as originally being `args[0]` (because why on earth would it be `1` requiring [this sillyness](https://github.com/mozilla-services/cloudops-infra/blob/9dc1f77/projects/sync/k8s/charts/tokenserver/templates/deployment-purge-old-records.yaml#L56) :rage1:)